source/infrastructure/lib/aws-instance-scheduler-stack.ts

```diff
@@ -15,22 +15,22 @@
  *****************************************************************************/
```

Comment:
Please replace the long copyright headers with the short ones as you encounter them.

Reply:
I would rather do this as a separate story/pr and change them all at once using an IDE macro